lightkurve/lightcurve.py

```diff
@@ -42,6 +42,9 @@ def __init__(self, time, flux, flux_err=None, meta={}):
             self.flux_err = np.nan * np.ones_like(self.time)
         self.meta = meta
 
+    def __repr__(self):
+        return('LightCurve Object')
+
```

Comment:
I think we don't need this for now

Reply:
I don't agree I think this is user friendly and a quick way to distinguish between the three classes of light curve that you can have.

Reply:
Right now it says `<lightkurve.lightcurve.LightCurve object at 0x7f3dd738a780>`.  You could argue that is more informative than `LightCurve Object`.

Reply:
We did argue, I think it's more useful for a normal user. I am worried that people will be confused when there are three possible objects that their light curve can be. I think printing a very simple 'LightCurve', 'KeplerLightCurve' or 'KeplerLightCurveFile' will help people debug why they can't run their favourite methods on their lightcurve.

Reply:
Do you mean that users will glance over `<lightkurve.lightcurve.LightCurve object at 0x7f3dd738a780>` because it looks scary, whereas `LightCurve Object` is more readable.  I think developers would deffo prefer the former, but I can see that new users may prefer the latter?

Reply:
Yeah that's exactly my point, it just makes it less scary for users. I reasoned that most of our users won't be developers. We should try to be as user friendly as possible, right?

Reply:
I think that's not the purpose of ``__repr__``. I still would prefer ``<lightkurve.lightcurve.LightCurve object at 0x7f3dd738a780>``.

reference:
https://stackoverflow.com/questions/1436703/difference-between-str-and-repr-in-python

Reply:
We're not losing information so much as improving readability. I still think this is more helpful to our average user. And if we're doing `__repr__` like this for all the other classes it makes sense they all look like this.